index.js

```diff
@@ -742,10 +768,11 @@ DiscordClient.on('message', message => {
       helpMessage += '3. `' + prefix + 'warstats #CLANTAG` Display war stats for a clan that is tracked by The Announcer. If not provided with a clan tag it will display war stats for all clans assigned to the channel the command was run in.\n'
       helpMessage += '4. `' + prefix + 'hitrate #CLANTAG` Display hit rate stats for a clan that is tracked by The Announcer. If not provided with a clan tag it will display hit rate stats for all clans assigned to the channel the command was run in.\n'
       helpMessage += '5. `' + prefix + 'playerstats #PLAYERTAG` Display player stats for any player tag provided.\n'
-      helpMessage += '6. `' + prefix + 'style [1-6](+)` Choose a style to use for war attacks in this channel. Requires a number to select style type, optionally append a `+` if you want war stats included in every message.\n'
+      helpMessage += '6. `' + prefix + 'style [1-8](+)` Choose a style to use for war attacks in this channel. Requires a number to select style type, optionally append a `+` if you want war stats included in every message.\n'
       helpMessage += '7. `' + prefix + 'filter all,attacks,defenses,none` Not yet implemented\n'
       // helpMessage += '7. `' + prefix + 'filter all,attacks,defenses,none` Filter which attacks show up in the channel.\n'
-      helpMessage += '8. `' + prefix + 'info` Display bot information.'
+      helpMessage += '8. `' + prefix + 'info` Display bot information.\n'
+      helpMessage += '9. `' + prefix + 'identify` bot as clan member (!identify #aaaaaaa).'
```

Comment:
Identification is required to allow wmbot to recognize that the announcer is a clanmember authorized to make commands